openapi/openapi.yaml

```diff
@@ -48,16 +42,13 @@ paths:
                 $ref: "#/components/schemas/Tool"
             text/plain:
               schema:
-                $ref: "#/components/schemas/Tool"
+                type: string
         "404":
           description: The tool can not be found.
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
-            text/plain:
```

Comment:
To confirm, this must be true, but the Dockstore implementation  always returns a JSON for errors?

Reply:
Not sure about how consistent Dockstore is, but I believe the TRS swagger intended it to be a JSON only